features/admin/impersonating_users.feature

```diff
@@ -0,0 +1,44 @@
+@user_impersonation
+Feature: Impersonating shop users
+    In order to provide topnotch customer support
+    As an Administrator
+    I want to impersonate a store user
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And there is an administrator "teddy@roosevelt.com"
+        And there is a customer "Tanith Low" identified by an email "brainsucker@london.uk" and a password "swordlover1918"
+        And I am logged in as "teddy@roosevelt.com" administrator
+
+    @ui
+    Scenario: Impersonating a customer
+        When I view details of the customer "brainsucker@london.uk"
+        And I try to impersonate them
+        Then I should be logged in as "Tanith Low"
+
+    @ui
+    Scenario: Inability to impersonate a customer with no account
+        Given there is a customer "Harold Zombie" identified by an email "harold@thrasher.ie" and a password "yesmaster"
```

Comment:
There is a customer with password? We should have a step where we create just customer (not ShopUser) without password.